src/Hooks.ts

```diff
@@ -16,14 +16,14 @@ export class _Hooks {
   /**
    * Process markdown before marked
    */
-  preprocess(markdown: string) {
+  preprocess(markdown: string): string | Promise<string> {
```

Comment:
Why `Promise<string>` here if its always returning a string?

Reply:
good catch! This was an early change